raster/rgb-renderer/README.md

```diff
@@ -1,31 +1,37 @@
-# Rgb Renderer
+# RGB renderer
 
-Apply an RGB renderer to a raster.
+Apply an RGB renderer to a raster layer to enhance feature visibility.
 
-An RGB renderer is used to adjust the color bands of a multispectral image.
+![Image of RGB renderer](RgbRenderer.png)
 
-![](RgbRenderer.png)
+## Use case
+
+An RGB renderer is used to adjust the color bands of a multispectral image. Remote sensing images acquired from satellites often contain values representing the reflection of multiple spectrums of light. Changing the RGB renderer of such rasters can be used to differentiate and highlight particular features that reflect light differently, such as different vegetation types, or turbidity in water.
 
 ## How to use the sample
 
-Choose one of the stretch parameter types. The other options will adjust based on the chosen type. Add your inputs and press the Update button to update the renderer.
+Choose one of the stretch parameter types. The other options will adjust based on the chosen type. Add your inputs and select the 'Render' button to update the renderer.
```

Comment:
In our sample the button is called 'Update'. I also think it's more correct to say 'click' rather than 'select' when talking about buttons:

```suggestion
Choose one of the stretch parameter types. The other options will adjust based on the chosen type. Add your inputs and click the 'Update' button to update the renderer.
```